### DISCUSSION TITLE
[v5] isAceCommand - feature not found in adonis v5

### DISCUSSION THREAD

**Discussion by iaamranjithkumar** [adonisjs/core#2932] [Help] [answered]
Hey Guys,

When running node ace migration:run, boot section of providers are also triggered. In another to avoid certain pieces of code not to get triggered when ace commands are run I was using isAceCommand.
 
Now code is migrated to adonis v5 and I don't find isAceCommand in Helpers or is there any new way of implementation to identify if the process is triggered by ace command? 

Regards,
Ranjith

**Comment by thetutlage** [comment, marked as answer]
You need to use the application environment to detect the environment in which the app is running. Relevant docs. https://docs.adonisjs.com/guides/application

**Reply by iaamranjithkumar** [reply]
@thetutlage   -Thank you.. Got it 👍
